client/java-armeria/src/main/java/com/linecorp/centraldogma/client/armeria/ArmeriaCentralDogma.java

```diff
@@ -801,12 +802,13 @@ private static PushResult push(AggregatedHttpResponse res) {
     @Override
     public CompletableFuture<Revision> watchRepository(String projectName, String repositoryName,
                                                        Revision lastKnownRevision, String pathPattern,
-                                                       long timeoutMillis) {
+                                                       WatchOptions watchOptions) {
```

Comment:
Sorry for making you do this twice but let's remove `WatchOptions`. 😅 
What I'm trying to add in [another PR](https://github.com/line/centraldogma/pull/651) is that a fluent style for `CentralDogma`:
```java
dogma.forRepo("foo", "bar")
     .watchingFiles("/**")
     .timeoutMillis(...)
     .errorOnEntryNotFound(...)
     .watch(revision);
```
If we do this, I think we no longer need `WatchOptions` because we can just add a method that takes the whole parameters and deprecate others:
```java
// We are going to use this method.
CompletableFuture<Revision> watchRepository(String projectName, String repositoryName,
                                            Revision lastKnownRevision, String pathPattern,
                                            long timeoutMillis, boolean errorOnEntryNotFound);

// Other methods are deprecated because users can use fluent style API.
// dogma.forRepo("foo", "bar").watchingFiles(pathPattern).watch(lastKnownRevision);
@Deprecated
CompletableFuture<Revision> watchRepository(String projectName, String repositoryName,
                                            Revision lastKnownRevision, String pathPattern);

```

Reply:
I'm interested in the fluent style. Also I think it is good idea than my changes.
I try remake codes base on your changes